src/SupervisionSuccess/components/StatePicker/StatePicker.js

```diff
@@ -16,6 +16,7 @@ const StatePicker = ({ states, state, onStateChange }) => {
     [onStateChange]
   );
 
+  // TODO: Need to know where from year need to come
```

Comment:
Is this TODO still in-progress?

Reply:
Yes, PRD does not describe where to get this data.

Reply:
Oh I think this number is actually meant to be hard-coded to 2017. Let's stick with it for now but I'll ping the PRD to ask.

Reply:
@jessex Andrew Warren updated spreadsheet and added year, so i will update code, but in another PR